workflow/modules/postprocess/Snakefile

```diff
@@ -98,7 +98,6 @@ rule subset_indels:
     input: 
         vcf = "results/{refGenome}/postprocess/{prefix}_filtered.TEMP.vcf.gz",
         idx = "results/{refGenome}/postprocess/{prefix}_filtered.TEMP.vcf.gz.csi",
-        ref = "results/{refGenome}/data/genome/{refGenome}.fna"
```

Comment:
Need to delete comma from line 100 I think, otherwise will error

Reply:
I think its OK. It passed the postprocess test. Trailing commas are typically OK ime

Reply:
right, I see that. I'll approve